doc/protocol.md

```diff
@@ -1066,6 +1066,11 @@ The following options can be specified in the "open" control message:
    you don't set this field, the actual tag will not be checked.  To
    express that you expect the file to not exist, use "-" as the tag.
 
+* "attrs": a JSON object containing file user and group information
+           to set. Both `user` and `group` are required.
```

Comment:
This is slightly misleading -- for now you can set both or neither, i.e. an empty attrs is permitted. We may also add further attributes in the future. So perhaps move this to the user/group documentation below, as "if you specify this, you also have to specify `group`" or so?

Reply:
The whole section is kinda out of date, needs rewording